script.js

```diff
@@ -395,37 +414,42 @@ document.addEventListener('DOMContentLoaded', async function () {
         currentPage = parseInt(event.target.value);
         displayOffers(currentPage);
     });
-
     function filterOffers() {
         currentSort = { column: null, order: 'asc' };
 
-        // Start with all offers
         let tempFilteredOffers = [...offers];
 
-        // Filter by search string if applicable
+        // Apply search string filter if applicable
```

Comment:
Can you please remove these comment changes and keep the overall changes in this PR as minimal as possible?